app/controllers/application_controller.rb

```diff
@@ -15,4 +15,8 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def after_sign_in_path_for(resource)
```

Comment:
Should we do logout path as well? If you log out, should it redirect you back to your previous page or is redirecting you to root_path logically in this case?